screens/Profile.tsx

```diff
@@ -1,155 +1,65 @@
-import { StyleSheet, Image, ScrollView } from 'react-native';
+import { StyleSheet, Image } from 'react-native';
 import { FontAwesome } from '@expo/vector-icons';
 import { Text, View } from '../components/Themed';
 import { Button } from 'react-native';
 import * as SecureStore from 'expo-secure-store';
 import { useAppDispatch } from '../store/App.hooks';
-import { AuthAction } from '../store/Auth.action';
+import { AuthAction } from '../store/actions/Auth.action';
+import {
+  Heading,
+  Body,
+  CenterText,
+  Container,
+  ScrollContainer,
+  StyledImage,
+} from '../components/Common';
 
 export default function ProfileScreen() {
-    const dispatch = useAppDispatch();
+  const dispatch = useAppDispatch();
 
+  const logout = async (): Promise<void> => {
+    await SecureStore.deleteItemAsync('auth-cookies');
+    await SecureStore.deleteItemAsync('user-id');
```

Comment:
remove this line

Reply:
It's already removed on the other branches but there's so many updates 😭